Yoast/wordpress-seo
# Adds dismiss all button to the notification page

## Summary

This PR can be summarized in the following changelog entry:

Adds a dismiss all button to the notification center. Also cleans up / de-dupes some of the code for the individual notifications.
## Test instructions

This PR can be tested by following these steps:
- Go to the notification center where there are still active notifications
- Click on the `Dismiss All Notifications` button.
- All notifications should now be dismissed.

Fixes #4870
